### ISSUE TITLE
Consider renaming facets to all singular or plural 

### ISSUE THREAD

**Issue by dialsunny** [GSA/datagov-wptheme#513]
http://catalog.data.gov/dataset

Facets are TopicS, TagS, FormatS, OrganisationS

Should Publisher also be called PublisherS 

![image](https://cloud.githubusercontent.com/assets/5984939/5004645/a8a37542-69ed-11e4-968f-c646ecb76d0c.png)

**Comment by rebeccawilliams** [comment]
Agreed consistency is a good call and this brings up a similar Issue as #409.

To this first point, I think singular is more precise. 

Usability testing has also indicated that these could be more conspicuously labeled as Filters, as the first one is _Filter by location_ is:

![screen shot 2014-11-20 at 12 26 40 pm](https://cloud.githubusercontent.com/assets/1725454/5129510/e08fecd2-70b0-11e4-98e6-bc5aeddb00f8.png)

Usability testing has also indicated that date (currently not available), location, jurisdiction (aka 'Organization Type') and agency ('Organizer', 'Publisher') are the most asked for filters.
